### PR TITLE
Add a requirements.txt pip file compatible with Python 3

### PR DESCRIPTION
**Pull request by dconnolly**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cffi==1.14.5
```
Comment (str4d) [inline requirements.txt:1]: This is not a direct dependency, but I confirmed that it is pulled in by `cryptography`, so I'm fine with it being pinned here.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+chacha20poly1305==0.0.3
+cryptography==3.4.7
+numpy==1.21.0
+pyblake2==1.1.2
+pycparser==2.20
```
Comment (str4d) [inline requirements.txt:6]: This is not a direct dependency, but I confirmed that it is pulled in by `cryptography`, so I'm fine with it being pinned here.

**Review by str4d — APPROVED** [review]
Tested ACK.

**Pull request merged**